tools/virtualization/fetching-ovmf-uefi-from-the-correct-source.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Fetching OVMF UEFI from the correct source  
+subsection: virtualization  
+order: 2  
```

Comment:
Why is this order 2?

Could we keep the order from the `about` document. IOW this would have preferably much higher order.

Reply:
*Or even move this to the example usage sub-page

 _ _ _ _

_Basically, what makes more sense to you, but I'd definitely give it some structure._

Reply:
I think it is okay for it to be had here separately. Come to think of it, it might be seen as an optional thing and people might be okay with just using BIOS boot while avoiding UEFI entirely.